lib/hawkular/inventory/entities.rb

```diff
@@ -119,6 +119,7 @@ class OperationDefinition < BaseEntity
     def initialize(op_hash)
       super(op_hash)
       @params = {}
+      return if op_hash['properties'].nil?
```

Comment:
On 17 Aug 2016, at 16:13, abonas wrote:

> maybe worth to check also `.empty?` ?  in case it's not nil but is an 
> empty hash for some reason.

If they are empty then the next check should catch
this.